src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -26,17 +34,98 @@ private void initMenu() {
         this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
 
         this.add(this.importMenu);
-        this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
+        this.add(this.settingsMenu);
+
         // importMenu should be disabled  before signing in
-        this.importMenu.setEnabled(false);
+//        this.importMenu.setEnabled(false);
+
+        // Add function to import menu
+        importMenu.addMenuListener(new MenuListener() {
+            @Override
+            public void menuSelected(MenuEvent e) {
+                importBills();
+            }
+
+            @Override
+            public void menuDeselected(MenuEvent e) {
+
+            }
+
+            @Override
+            public void menuCanceled(MenuEvent e) {
+
+            }
+        });
+        this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
+
+        // Add function to setting menu
+        settingsMenu.addMenuListener(new MenuListener() {
+            @Override
+            public void menuSelected(MenuEvent e) {
+                System.out.println("yes");
+                int[] setting = setting();
+
+                MainFrame.getBillPanel().changeColor(colors[setting[0]]);
+                MainFrame.getActionPanel().changeColor(colors[setting[0]]);
+
+                MainFrame.getActionPanel().changeFont(fonts[setting[1]]);
+                MainFrame.getBillPanel().changeFont(fonts[setting[1]]);
+
+            }
+
+            @Override
+            public void menuDeselected(MenuEvent e) {
+
+            }
+
+            @Override
+            public void menuCanceled(MenuEvent e) {
+
+            }
+        });
+
 
-        this.add(this.settingsMenu);
         this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
 
         this.add(this.helpMenu);
     }
 
+    // Function to set up the setting dialog
+    private int[] setting() {
+        SettingDialog setdlg = new SettingDialog(this);
```

Comment:
We should then use `SwingUtil.centerInOwner` to display the setting dialog in the center of our main window.